src/assets/scss/custom.scss

```diff
@@ -1,7 +1,7 @@
 $primary: #404F65;
 $secondary: #D65F30;
 $success: #219653;
-$info: #032DF2;
+$info: #3144EC;
```

Comment:
Looks like changes from the previous PR still persists in this PR. Please clean up by removing the already merged code.

Reply:
Rebased.